primitives/src/lib.rs

```diff
@@ -5,8 +5,8 @@ use codec::{Decode, Encode};
 use scale_info::TypeInfo;
 pub use v0::*;
 
-/// Identifier for the class of asset.
-pub type AssetId = u32;
+/// The identifier of the token.
```

Comment:
```suggestion
/// The identifier of a token.
```
Only as its a public comment.